llama-index-integrations/tools/llama-index-tools-azure-dynamic-sessions/README.md

```diff
@@ -0,0 +1,49 @@
+# Azure Dynamic Sessions Tool
+
+This tool leverages Azure Dynamic Sessions Pool to enable an Agent to run generated Python code in a secure environment with very low latency.
+
+In order to utilize the tool, you will need to have the Session Pool management endpoint first. More details in here: "some-url"
+
+## Usage
+
+A more detailed sample is located in a Jupyter notebook [here](https://github.com/run-llama/llama_index/tree/main/llama-index-integrations/tools/llama-index-tools-azure-dynamic-sessions/examples/azure_dynamic_sessions.ipynb)
+
+Here's an example usage of the `AzureDynamicSessionsToolSpec`.
+
+```python
+from llama_index.tools.azure_dynamic_sessions import (
+    AzureDynamicSessionsToolSpec,
+)
+from llama_index.core.agent import ReActAgent
+from llama_index.llms.azure_openai import AzureOpenAI
+
+llm = AzureOpenAI(
+    model="gpt-35-turbo",
+    deployment_name="gpt-35-deploy",
+    api_key=api_key,
+    azure_endpoint=azure_endpoint,
+    api_version=api_version,
+)
+
+dynamic_session_tool = AzureDynamicSessionsToolSpec(
+    pool_managment_endpoint="your-pool-management-endpoint"
+)
+
+agent = ReActAgent.from_tools(
+    dynamic_session_tool.to_tool_list(), llm=llm, verbose=True
+)
+
+print(agent.chat("Tell me the current time in Seattle."))
```

Comment:
what would this prompt actually execute? 👀

Reply:
ah I'll include the results returned from the prompt in the next commit.

Reply:
@logan-markewich btw I see that the sample jupyter notebook is removed from this PR. Is there a specific place to put the sample? The readme file is currently has a link that points to the same directory as if the sample notebook is merged into main branch in this repo

Reply:
Ah sorry, I moved it to our proper examples folder, but forgot to `git add` it, will add that back!

Reply:
Added back!